hiss/wave/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
Also, a directory structure like:

```
application/
    tests/
        __init__.py
        model_tests/
            __init__.py
            wave.py
```

```python
# application/tests/model_tests/__init__.py

from .wave import *
```

```python
# application/tests/__init__.py

from .model_tests import *
```

will help make testing easier in the future.